etc/config/c++.amazon.properties

```diff
@@ -2243,13 +2243,18 @@ compiler.mipsg1320.demangler=/opt/compiler-explorer/mips/gcc-13.2.0/mips-unknown
 
 ## MIPS 64
 group.mips64.groupName=MIPS64 GCC
-group.mips64.compilers=mips64g1210:mips64g1220:mips64g1230:mips64g1310:mips64g1320:mips564:mips112064
+group.mips64.compilers=mips64g1210:mips64g1220:mips64g1230:mips64g1310:mips64g1320:mips564:mips112064:mips64g1050
```

Comment:
small nit: can you keep the (kind) of order we have here? Here, the list is ordered by size (32/64) and then by versions.
That's a very small change, but when we introduce a new GCC version on the 20+ targets we have, it's harder if every target has its own ordering/naming.